### ISSUE TITLE
No pop-up first video

### ISSUE THREAD

**Issue by hockeymachine86** [im85288/service.upnext#197]
When I play a video (local library), I get no pop-up the first time I open kodi to watch.  Meaning that if I open kodi and start watching a series, no pop-up.  But if I manually select the next episode in the series, when I get to the end of that episode, I get an episode.

From the log: 
2020-06-30 20:07:48.629 T:7497  - Start watching episode 
2020-06-30 20:52:29.007 T:7497  - I have to manually open the next episode
Log Part 1: https://pastebin.com/sczn9FPx
Log Part 2: https://pastebin.com/fG2rRt2c

**Comment by dagwieers** [comment]
I think this is because Kodi does not know what the season and episode is because it does not understand the formatting. You probably have to reconfigure Kodi to make it understand 3x19, or rename your files to use the S02E05 convention.



This is not an Up Next bug.

**Comment by hockeymachine86** [comment]
Thanks. Is there a reason it shows up after the first video plays. Meaning if I watch 3x19, it doesn't popup, but if I open 3x20, it will popup when that one ends for me to continue onto 3x21?

**Comment by dagwieers** [comment]
No, it should either work for all, or not work at all. And that is how it behaves for me with local video too. Only when Kodi doesn't know what the season/episode is, or did not determine that the next episode is available it will not show a pop-up notification. In your case it should if both requirements are met.

**Comment by hockeymachine86** [comment]
Okay, so the issue I have is it doesn't work for the first video but then works onward.

**Comment by hockeymachine86** [comment]
Wondering if it's because it doesn't know it exists. Am playing from a samba share. I have experienced the same from shield and firestick for reference.

**Comment by dagwieers** [comment]
And this is the case for every series and every season? Or just in this particular case?
Also, episode 19 is not the first episode, so this is confusing.

**Comment by dagwieers** [comment]
Are you playing from the Series interface in Kodi, or just from a video-listing?
Because, as I said, Kodi needs to know the season/episode of the file being played, and needs to know the next episode exists.
If you play from a video listing, it doesn't know any of this information, unless some files have been processed.

So I still don't think this is an Up Next issue, but related to how you have added files to Kodi. (Some have been indexed, others have not?)

**Comment by hockeymachine86** [comment]
It's playing from a series listing. Yes happens with all series and seasons/episodes. Right episode 19 is the example of last watched episode. Each show is in its own folder and was added to the library as a TV show with each folder containing only a single TV show.

**Comment by dagwieers** [comment]
Please test the upcoming v1.1.3 release by testing PR #217 and reporting back.

**Comment by hockeymachine86** [comment]
Will do. I'm new to github.how do I download the files to test them?

**Comment by dagwieers** [comment]
@hockeymachine86 You click on the branch, and then press on `Code` » `Download as ZIP`
https://github.com/dagwieers/service.upnext/archive/prepare-for-1.1.3.zip

This ZIP file needs to be downloaded/transferred to your device and then needs to be installed as ZIP in Kodi.
